### DISCUSSION TITLE
Run actions on private (security) fork from a public repository

### DISCUSSION THREAD

**Discussion by frankcorneliusmartin** [community/community#34198] [Actions]
We have a public repository with some actions. These actions are normally run on a pull request. Now we solved an issue through the security advisor which created a private fork from the repository to solve the issue. The problem is that the checks (actions) do not run. Thereby blocking the merging process. 

Ofcourse I could bypass this check but I am looking for a better solution.

**Comment by hansmbakker** [comment]
> which created a private fork from the repository

How did you create a private fork of a public repository? I believe that's not possible? Also see https://github.com/community/community/discussions/34584

**Reply by frankcorneliusmartin** [reply]
You can create private forks from a public repository through the secutiry advisor to patch security vulnerabilities.
